client/extensions/woocommerce/store-sidebar/index.js

```diff
@@ -117,6 +117,32 @@ class StoreSidebar extends Component {
 		);
 	}
 
+	reviews = () => {
+		if ( ! config.isEnabled( 'woocommerce/extension-reviews' ) ) {
+			return null;
+		}
+
+		const { site, siteSuffix, translate } = this.props;
+		const link = '/store/reviews' + siteSuffix;
+		const selected = this.isItemLinkSelected( [ link ] );
+		const classes = classNames( {
+			reviews: true,
+			'is-placeholder': ! site,
+			selected,
+		} );
+
+		// TODO Add bubble containing count of unapproved reviews.
+		return (
+			<SidebarItem
+				className={ classes }
+				icon="star-outline"
+				label={ translate( 'Reviews' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'Latest Reviews' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).